### DISCUSSION TITLE
Privacy Rules in Germany

### DISCUSSION THREAD

**Discussion by mendikant** [poblabs/weewx-belchertown#864] [Q&A]
Here in Germany we have strict privacy rules for website operators 

According to them it is not possible to use the skin here because Google Fonts and Cloudflare things are downloaded.
There is a risk of a warning from a lawyer, which could be expensive.
Is there a possibility to integrate that locally in the skin?

I found a good implementation here: https://www.woellsdorf-wetter.de/imprint/

Regards Mendikant

**Comment by c0by85** [comment]
Check here. There is already a solution.

https://github.com/poblabs/weewx-belchertown/discussions/810
